irohad/model/execution/impl/command_executor.cpp

```diff
@@ -541,9 +548,15 @@ namespace iroha {
     bool SetQuorumExecutor::isValid(const Command &command,
                                     ametsuchi::WsvQuery &queries) {
       auto set_quorum = static_cast<const SetQuorum &>(command);
+      auto signatories = queries.getSignatories(set_quorum.account_id);
 
-      // Quorum must be from 1 to N
-      return set_quorum.new_quorum > 0 and set_quorum.new_quorum < 10;
+      if (not(signatories.has_value())) {
+        // No  signatories of an account found
```

Comment:
how can this case ever happen? If an account has no signatories, then it cannot send a transaction. delete it